src/Block/Service/MenuBlockService.php

```diff
@@ -130,100 +105,41 @@ public function getMetadata(): MetadataInterface
         ]);
     }
 
-    /**
-     * @return array<array{string, class-string<FormTypeInterface>, array<string, mixed>}>
-     */
-    private function getFormSettingsKeys(): array
+    protected function getFormSettingsKeys(): array
     {
         $choiceOptions = [
             'required' => false,
-            'label' => 'form.label_url',
+            'label' => 'form.label_menu_name',
             'choice_translation_domain' => 'SonataBlockBundle',
```

Comment:
I'm not sure the choice_translation_domain must be SonataBlockBundle here, since the alias name are coming from `$this->menuRegistry->getAliasNames()` so this are alias from the user.

Reply:
I would say to touch this topic on another PR if any. I guess this has been like this for ages.